tests/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -559,6 +559,36 @@ public function testListForeignKeys(): void
         self::assertEquals('CASCADE', $fkeys[0]->getOption('onDelete'));
     }
 
+    public function testCreateForeignKeyGh5485(): void
+    {
+        if (! $this->connection->getDatabasePlatform()->supportsForeignKeyConstraints()) {
+            self::markTestSkipped('Does not support foreign key constraints.');
```

Comment:
this PR is a fix so it should be merged into v3.3.x

modified unit tests are passing

to test this functional test /w Sqlite, this line needs to be removed/commented out, in v3.4.x `supportsForeignKeyConstraints` is `true`, so since v3.4.x this functional test will test Sqlite implicitly

Reply:
Let's target 3.4.x then. On 3.3.x, this test is not executed so we cannot test your change.

Reply:
As long as 3.4.x will be released soon... retargetted.